site/docs/capabilities/observability/tracing.md

```diff
@@ -0,0 +1,139 @@
+---
+id: tracing
+title: GenAI Distributed Tracing
+sidebar_position: 7
+---
+
+Envoy AI Gateway's router joins and records distributed traces when supplied
+with an [OpenTelemetry](https://opentelemetry.io/) collector endpoint.
+
+This guide provides an overview of the spans recorded by the AI Gateway and how
+export them to your choice of OpenTelemetry collector.
+
+## Overview
+
+Envoy AI Gateway's router joins and records distributed traces when supplied
+with an [OpenTelemetry](https://opentelemetry.io/) collector endpoint.
+
+Requests to the OpenAI Chat Completions endpoint are recorded as Spans which
+include typical timing and request details. In addition, there are GenAI
+attributes representing the LLM call including full request and response
+details, defined by the [OpenInference semantic conventions][openinference].
+
+OpenInference attributes default to include full chat completion request and
+response data. This can be toggled with configuration, but when enabled allows
+systems like [Arize Phoenix][phoenix] to perform LLM evaluations of production
+requests captured in OpenTelemetry spans.
+
+## Trying it out
+
+Before you begin, you'll need to complete the basic setup from the
+[Basic Usage](/docs/getting-started/basic-usage) guide, which includes
+installing Envoy Gateway and AI Gateway.
+
+### Install Phoenix for LLM observability
+
+```shell
+# Install Phoenix using SQLite
```

Comment:
@mikeldking I used this since Colima doesn't play nice with PVCs etc and it is a demo anyway. If you see anything wrong here, lemme know or suggest an edit!